docs/products/flink/concepts/important_concepts.rst

```diff
@@ -0,0 +1,42 @@
+Important concepts in Apache Flink
+==================================
+
+This article gives you a brief explanation of some of the relevant terms and concepts in Apache Flink. A basic understanding of these helps you get the most out of our other Apache Flink articles.
+
+
+Event and processing times
+--------------------------
+
+Event time refers to when events actually happen, while processing time refers to when events are observed within a system. Various factors affect how events are processed, including shared hardware resources and network congestion, distributed system logic, and variances in the data throughput and order. Because of this, there can be significant differences between the event time and processing time for an event, even though ideally they would be equal.
+
+In Apache Flink, the streamed data does not always arrive in the same order as the events occurred. This means that using processing time in your applications can cause issues in system behavior. For this reason, we recommend that you use event time to process data, as it allows your applications to maintain the correct event sequence throughout the data streaming pipeline.
+
+For more information, see the `Apache Flink documentation <https://ci.apache.org/projects/flink/flink-docs-release-1.13/docs/concepts/time/>`_.
+
+
+Windows
+-------
+
+Apache Flink uses *windows* to split the streamed data into segments that can be processed. Due to the nature of data streams, there is never a situation where *all* of the data is available, because you would be waiting indefinitely for new data points to arrive - so instead, windowing offers a way to define a subset of data points that you can then process and analyze.
```

Comment:
Should we add "Due to the **unbounded** nature of data streams"?

Reply:
I was hoping to avoid using "unbounded" (because it somehow doesn't look like correct English), but you have a point, so that's now included ;)